Source/WebGPU/WebGPU/ComputePassEncoder.mm

```diff
@@ -39,8 +39,8 @@
 
 #define RETURN_IF_FINISHED() \
 if (!m_parentEncoder->isLocked() || m_parentEncoder->isFinished()) { \
-    m_device->generateAValidationError([NSString stringWithFormat:@"%s: failed as encoding has finished", __PRETTY_FUNCTION__]); \
-    return; \
+m_device->generateAValidationError([NSString stringWithFormat:@"%s: failed as encoding has finished", __PRETTY_FUNCTION__]); \
+return; \
```

Comment:
Not too important, but we should probably maintain the indent on these two lines.

Reply:
oh thanks for catching, that was changed by mistake, I will revert